src/Mod/Path/PathScripts/PathPreferences.py

```diff
@@ -29,35 +29,38 @@
 # PathLog.setLevel(PathLog.Level.INFO, PathLog.thisModule())
 # PathLog.trackModule()
 
-DefaultFilePath           = "DefaultFilePath"
-DefaultJobTemplate        = "DefaultJobTemplate"
-DefaultStockTemplate      = "DefaultStockTemplate"
-DefaultTaskPanelLayout    = "DefaultTaskPanelLayout"
-
-PostProcessorDefault      = "PostProcessorDefault"
-PostProcessorDefaultArgs  = "PostProcessorDefaultArgs"
-PostProcessorBlacklist    = "PostProcessorBlacklist"
-PostProcessorOutputFile   = "PostProcessorOutputFile"
-PostProcessorOutputPolicy = "PostProcessorOutputPolicy"
-
-LastPathToolBit           = "LastPathToolBit"
-LastPathToolLibrary       = "LastPathToolLibrary"
-LastPathToolShape         = "LastPathToolShape"
-LastPathToolTable         = "LastPathToolTable"
-
-LastFileToolBit           = "LastFileToolBit"
-LastFileToolLibrary       = "LastFileToolLibrary"
-LastFileToolShape         = "LastFileToolShape"
-
-UseLegacyTools            = "UseLegacyTools"
-UseAbsoluteToolPaths      = "UseAbsoluteToolPaths"
-OpenLastLibrary           = "OpenLastLibrary"
+DefaultFilePath             = "DefaultFilePath"
+DefaultJobTemplate          = "DefaultJobTemplate"
+DefaultStockTemplate        = "DefaultStockTemplate"
+DefaultTaskPanelLayout      = "DefaultTaskPanelLayout"
+
+PostProcessorDefault        = "PostProcessorDefault"
+PostProcessorDefaultArgs    = "PostProcessorDefaultArgs"
+PostProcessorBlacklist      = "PostProcessorBlacklist"
+PostProcessorOutputFile     = "PostProcessorOutputFile"
+PostProcessorOutputPolicy   = "PostProcessorOutputPolicy"
+
+LastPathToolBit             = "LastPathToolBit"
+LastPathToolLibrary         = "LastPathToolLibrary"
+LastPathToolShape           = "LastPathToolShape"
+LastPathToolTable           = "LastPathToolTable"
+
+LastFileToolBit             = "LastFileToolBit"
+LastFileToolLibrary         = "LastFileToolLibrary"
+LastFileToolShape           = "LastFileToolShape"
+
+UseLegacyTools              = "UseLegacyTools"
+UseAbsoluteToolPaths        = "UseAbsoluteToolPaths"
+OpenLastLibrary             = "OpenLastLibrary"
 
 # Linear tolerance to use when generating Paths, eg when tessellating geometry
-GeometryTolerance         = "GeometryTolerance"
-LibAreaCurveAccuracy      = "LibAreaCurveAccuarcy"
+GeometryTolerance           = "GeometryTolerance"
+LibAreaCurveAccuracy        = "LibAreaCurveAccuarcy"
 
-EnableExperimentalFeatures = "EnableExperimentalFeatures"
+WarningSuppressRapidSpeeds  = "WarningSuppressRapidSpeeds"
+WarningSuppressSelectionMode = "WarningSuppressSelectionMode"
```

Comment:
I think if you're going to tweak the spacing on all the above lines, may as well add one more to make line 61 line up, too.